docs/content/custom-annotations.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Custom Annotations
+
+description: 
+weight: 100
+doctypes: ["concept"]
+toc: true
+---
```

Comment:
We can remove this doc and add an alias to the frontmatter of the doc in the new location.

Reply:
done